week-4-practise/level-2/exercise-1.js

```diff
@@ -10,13 +10,17 @@ We made a start on this for you here:
 
 var bottle = {
   volume: 0,
-  fill: function() {
+  fill: function () {
+    return this.volume = 100;

```

Comment:
You don't actually need to return a value here.

Reply:
Will commit the change